src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -686,36 +687,47 @@ tohexOrText s = case "0x" `Char8.isPrefixOf` encodeUtf8 s of
 vmFromCommand :: Command Options.Unwrapped -> IO EVM.VM
 vmFromCommand cmd = do
   withCache <- applyCache (state cmd, cache cmd)
-  vm <- case (rpc cmd, address cmd, code cmd) of
+
+  (miner,ts,blockNum,diff) <- case rpc cmd of
+    Nothing -> return (0,0,0,0)
```

Comment:
```suggestion
    Nothing -> return ( fromMaybe 0 (coinbase cmd)
                      , fromMaybe 0 (timestamp cmd)
                      , fromMaybe 0 (number cmd)
                      , fromMaybe 0 (difficulty cmd)
                      )
```

Reply:
These vars are only the default values for all these params. See `word`  function in vm0